sr_utilities/scripts/sr_utilities/trajectory_controller_spawner.py

```diff
@@ -131,13 +132,49 @@ def set_controller(self):
         if not success:
             rospy.logerr("Failed to launch trajectory controller!")
 
+    @staticmethod
+    def wait_for_topic(topic_name, timeout):
```

Comment:
This function is basically taken from https://github.com/ros-controls/ros_control/blob/indigo-devel/controller_manager/scripts/spawner#L141